charts/tidepool/charts/abbott/templates/0-configmap.yaml

```
@@ -0,0 +1,19 @@
+{{ if .Values.configmap.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: abbott
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "charts.labels.standard" . }}
+data:
+{{ if .Values.configmap.redirectURL }}
+  RedirectURL: {{ .Values.configmap.redirectURL }}
+{{ else }}
+  RedirectURL: "{{include "charts.host.api" .}}/v1/oauth/abbott/redirect"
+{{ end }}
+  TokenURL: {{ .Values.configmap.tokenURL | default "https://oauth.libreview.io/oauth2/token" }}
+  AuthorizeURL: {{ .Values.configmap.authorizeURL | default "https://oauth.libreview.io/oauth2/auth" }}
+  ClientURL: {{ .Values.configmap.clientURL | default "https://api.libreview.io" }}
```

Why don't we just remove the defaults here.
```suggestion
  TokenURL: {{ .Values.configmap.tokenURL }}
  AuthorizeURL: {{ .Values.configmap.authorizeURL }}
  ClientURL: {{ .Values.configmap.clientURL }}
```
We'll just need to ensure they are specified in the cluster config files.